thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -1011,7 +1012,8 @@ private String getDimensions(String dataset, String exploreDimensions) throws Ex
 
       for (MergedAnomalyResultDTO mergedAnomaly : mergedResults) {
         // if use notified flag, only keep anomalies isNotified == true
-        if ( (useNotified && mergedAnomaly.isNotified()) || !useNotified) {
+        if ( (useNotified && mergedAnomaly.isNotified()) || !useNotified || mergedAnomaly.getAnomalyResultSource().equals(
```

Comment:
Use AnomalyResultSource.USER_LABELED_ANOMALY).equals(...) instead.

Reply:
fixed. thanks for advising!